ktor-client/ktor-client-plugins/ktor-client-json/common/src/io/ktor/client/plugins/json/JsonPlugin.kt

```diff
@@ -111,6 +129,29 @@ public class JsonPlugin internal constructor(
         public fun receive(matcher: ContentTypeMatcher) {
             _receiveContentTypeMatchers += matcher
         }
+
+        /**
+         * Adds a type to the list of types that should be ignored by [ContentNegotiation].
+         *
+         * The list contains the [HttpStatusCode], [ByteArray], [String] and streaming types by default.
+         */
+        public inline fun <reified T> ignoreType() {
```

Comment:
does it make sense at all to add new functionality to deprecated plugin?

Reply:
Yep, we have internal breaking changes because of that